snapcraft.yaml

```diff
@@ -1447,8 +1447,15 @@ parts:
       fi
 
       # Setup bash completion
+      # Snap completer requires access to unaliased lxc binary
+      cp "${CRAFT_PART_INSTALL}/bin/lxc" "${CRAFT_PART_INSTALL}/bin/lxd.lxc"
```

Comment:
i dont think we should be doing this.

Reply:
@kadinsayani please can you scheduler a meeting with yourself @simondeziel and @masnax to discuss this, as I need to understand why the old one worked and why this one needs all these, dare I say it, "hacks".